elements/pfe-accordion/BaseAccordionHeader.scss

```diff
@@ -0,0 +1,135 @@
+:host { 
+    --pf-c-accordion__toggle--PaddingTop: var(--pf-global--spacer--sm, 0.5rem);
```

Comment:
Also, and I haven't thought this through entirely so disregard if you have a position on this, but we should perhaps move most of these to pfe-accordion.scss, especially if RHDS is going to override them.

Reply:
So you're thinking like instead of setting the var on host in the scss we would do something like 

```css
padding: var(--pf-global--spacer--sm, 0.5rem) var(--pf-global--spacer--md, 1rem) v var(--pf-global--spacer--sm, 0.5rem) var(--pf-global--spacer--md, 1rem);
```

or maybe something like 

```css
padding: var(--pf-c-accordion__toggle--PaddingTop, var(--pf-global--spacer--sm, 0.5rem)) var(--pf-c-accordion__toggle--PaddingRight, var(--pf-global--spacer--md, 1rem)) var(--pf-c-accordion__toggle--PaddingBottom, var(--pf-global--spacer--sm, 0.5rem)) var(--pf-c-accordion__toggle--PaddingLeft, var(--pf-global--spacer--md, 1rem));

```

Where we don't automatically set the variables but like you said we build off of the theming and then they can overwrite either all through the --pf-global styles or individually through the variables

Reply:
@markcaron do you know if these vars are meant to be user accessible? Is it ok if the user's preferences for these values from higher up the DOM tree are ignored?

Once this question is resolved, i'm good to merge